packages/abstractions/package.json

```diff
@@ -2,21 +2,25 @@
     "name": "@microsoft/kiota-abstractions",
     "version": "1.0.0-preview.1",
```

Comment:
my guess is the corresponding package.lock has been deleted for two reasons:
- switch to yarn
- lerna has a single lock at the root
But I just wanted to confirm?

Reply:
Yeah right. We do a single yarn install at the root where the locked versions are maintained